db/migrate/20201222123211_remove_unused_sign_in_related_columns.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveUnusedSignInRelatedColumns < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :authentication_tokens, :authenticable_id, :bigint
+    remove_column :authentication_tokens, :authenticable_type, :string
```

Comment:
Just curious, I guess the `remove_column` will remove the `index_authentication_tokens_on_id_and_type` index automatically? (It must do otherwise it would trigger a DB error).

Reply:
Yea it does 👍